.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1518,7 +1518,7 @@ jobs:
       - name: Login to GCR
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.REGISTRY_URL }}
+          registry: ${{ env.REGISTRY_URL }}
           username: _json_key
           password: ${{ secrets.GCR_CONFIG }}
```

Comment:
The first bug was caused when github actions was trying to reference a wrong variable. This address the first issue

Reply:
[use the env specified registry](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/3fa7ed650d218b687beb2cfe674a8067372dab4d)